app/models/response.rb

```diff
@@ -4,4 +4,16 @@ class Response < ActiveRecord::Base
 
   validates :poll_id, presence: true
   validates :choice_id, presence: true
+
+  def question
+    self.choice.question.text
+  end
+
+  def text
+    self.choice.text
+  end
+
+  def question_choices
+    self.choice.question.choices
+  end
```

Comment:
Damian I can't figure out how to obtain a specific poll question's response, when working from questions  to responses through choices. (Originally I did everything by going clockwise through our schema, starting from responses to questions through choices, I can't figure out how to go counterclockwise from questions to responses through choices.) Am I missing something?